manifests/exception.pp

```diff
@@ -135,8 +139,13 @@
     # Validate common parameters
     validate_re($ensure,['^(present|absent)$'])
     validate_slength($display_name,255)
-    validate_re($enabled,['^(yes|no)$'])
-    validate_re($allow_edge_traversal,['^(yes|no)$'])
+    # Cast enabled/allow_edge_traversal to strings, so we can have string
+    # values, as well as Bool (true/false), therefore we need to turn off
+    # the lint warning about only one variable in string...
+    # lint:ignore:only_variable_string
+    validate_re("${enabled}",['^(yes|no|true|false)$'])
```

Comment:
This looks like a great opportunity to move to puppet 4+ data types.  I don't think we should be updating any validate_re functions anymore.

@ofalk It's ok to just convert the ones you're interested in.  You wouldn't have to do the whole module, or even every parameter in this class. For `enabled`, you probably want something like...
```puppet
Variant[Enum['yes', 'no', 'true', 'false'], Boolean] $enabled = 'yes'
```
See https://puppet.com/docs/puppet/5.3/lang_data_abstract.html#examples-1